### ISSUE TITLE
Support EBS in ASG module.

### ISSUE THREAD

**Issue by Magicloud** [fpco/terraform-aws-foundation#245]
For instances in ASG that require storage other than `root`, terraform aws provider supports `ebs_block_device`. But `asg` module does not yet.

Ref: https://www.terraform.io/docs/providers/aws/r/launch_configuration.html#ebs_block_device

**Comment by bitemyapp** [comment]
This might be fixed by #249 

@qrilka Can you confirm?

**Comment by qrilka** [comment]
Yes, exactly, see https://github.com/fpco/terraform-aws-foundation/pull/249/files#diff-673c119ce48fb7e8788412a6ccdcdb58R98
